app/src/main/java/org/coepi/android/ui/extensions/ProgressBarViewExtension.kt

```diff
@@ -0,0 +1,24 @@
+package org.coepi.android.ui.extensions
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import org.coepi.android.R
+
+
+class ProgressBarViewExtension : LinearLayout {
+
+    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
+        orientation = HORIZONTAL
+        gravity = Gravity.CENTER
+
+        val inflater = context
+            .getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+        inflater.inflate(R.layout.progress_indicator, this, true)
+
+        this.isClickable = true
+        this.elevation = 10000.0f
```

Comment:
It doesn't seem necessary to add this. Let the developer determine what it overlays by the only the position in XML.

Reply:
Should I make this change in the XML? Because if I don't explicitly define the elevation, then the buttons will always lay on top (which i believe is the issue I was running with earlier commits). I thought XML views were layered on top of each other in the order of XML definitions, but since API 21, buttons are on top unless otherwise stated, which was allowing button clicks even though there was an overlay.

Reply:
Okay. I've used an overlay window in other projects, which doesn't have this problem. Trying out this approach here, since it seems more flexible. Let's keep the elevation then, but can we put it in a resource (such that it's also usable from XML), e.g. `elevation_max`, similar to here: https://stackoverflow.com/questions/19297522/android-integer-from-xml-resource so it's less likely that we run in conflicts with other parts setting "arbitrary large value" too?